src/lib/routers/history.ts

```diff
@@ -23,38 +54,53 @@ function defaultParseURL({ qsModule, location }) {
   //   - https://github.com/ljharb/qs#parsing-arrays
   //   - https://www.algolia.com/doc/api-reference/api-parameters/maxValuesPerFacet/
   return qsModule.parse(location.search.slice(1), { arrayLimit: 99 });
-}
+};
 
-function setWindowTitle(title) {
+const setWindowTitle = (title?: string): void => {
```

Comment:
no advantage having this as arrow IMO, but it's changed now so no big deal

Reply:
It was to be consistent with the two above functions.